app/attributes/attributedata.cpp

```diff
@@ -93,6 +93,23 @@ void FormItem::setConstraintHardValid( bool constraintHardValid )
   mConstraintHardValid = constraintHardValid;
 }
 
+FormItem::FormItemState FormItem::state() const
+{
+  return mState;
+}
+
+void FormItem::setState( FormItemState state )
+{
+  // We are currently using the state only for numeric fields,
+  // thus we here check for field type. We can use this functionality
+  // in future for other fields too, however, now we do not want
+  // to break working features, since this could prohibit user from saving the form
+  if ( mField.isNumeric() || mEditorWidgetSetup.type() == QStringLiteral( "Range" ) )
+  {
+    mState = state;
+  }
```

Comment:
not sure how to break existing features ? setState should be called with the Valiid for anything other. 

I would not be this paranoid :)

Reply:
not fixed yet?